### PR TITLE
chore: merge 1.33 from main to 2.0 feature branch

### PR DESCRIPTION
**Pull request by lengau**
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

**Comment by tigarmo** [comment]
@lengau can you merge `main` again? Should fix the tests

File: craft_parts/errors.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# along with this program.  If not, see <<http://www.gnu.org/licenses/>.
```
Comment (cmatsuoka) [inline craft_parts/errors.py:15]: There's a double `<` here.

**Review by cmatsuoka — APPROVED** [review]
No obvious anomalies detected.

File: craft_parts/errors.py (hunk 1 above)
Comment (lengau) [reply craft_parts/errors.py]: Thanks! I must have typoed that when I was looking for the branch difference in that file (the only conflict). Fixed!

**Pull request merged**